crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -718,17 +718,25 @@ where
         srcmap!(self, num, true);
 
         if self.cfg.minify {
-            value = minify_number(num.value);
-            self.wr.write_str_lit(DUMMY_SP, &value)?;
+            if num.value.is_infinite() && num.raw.is_some() {
+                self.wr.write_str_lit(DUMMY_SP, &num.raw.clone().unwrap())?;
```

Comment:
```suggestion
                self.wr.write_str_lit(DUMMY_SP, num.raw.as_ref().unwrap())?;
```

(in case this works)